packages/cdktf-cli/bin/cmds/helper/terraform-login.ts

```diff
@@ -114,12 +114,16 @@ the following file for use by subsequent Terraform commands:
   }
 
   public async getTerraformCredentialsFile(): Promise<TerraformCredentialsFile> {
-    const credentialsFile = JSON.parse(
-      fs.readFileSync(terraformCredentialsFilePath).toString()
-    );
-    const terraformCredentials: TerraformCredentialsFile = credentialsFile;
+    try {
+      const credentialsFile = JSON.parse(
+        fs.readFileSync(terraformCredentialsFilePath).toString()
+      );
+      const terraformCredentials: TerraformCredentialsFile = credentialsFile;
 
-    return terraformCredentials;
+      return terraformCredentials;
+    } catch (e) {
+      return { credentials: {} };
```

Comment:
we should at least add a trace / debug log here to debug this if the future might bring errors with weird setups once more 😅